client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/IndicesClientDocumentationIT.java

```diff
@@ -317,10 +317,10 @@ public void testCreateIndex() throws IOException {
                 message.put("type", "text");
                 Map<String, Object> properties = new HashMap<>();
                 properties.put("message", message);
-                Map<String, Object> tweet = new HashMap<>();
-                tweet.put("properties", properties);
-                jsonMap.put("tweet", tweet);
-                request.mapping("tweet", jsonMap); // <1>
+                Map<String, Object> _doc = new HashMap<>();
```

Comment:
nit: mapping instead of `_doc` ?

Reply:
Thanks @jimczi for catching this, my find-replace was overzealous :)